plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -74,14 +74,19 @@ class NewCreateServiceModal extends Component {
     const isEdit = location.pathname.includes('/edit');
     const serviceID = decodeURIComponent(params.id || '/');
     const service = isEdit ? DCOSStore.serviceTree.findItemById(serviceID) : null;
+    const isSpecificVersion = service instanceof Application && params.version;
```

Comment:
Yup, only `Application` has the `getVersions` method. Wanted to make sure that was the case before we called it. `Service` does not, and this could also be a `ServiceTree` (group).